examples/polypharmacy_gripnet/main.py

```diff
@@ -1,73 +1,60 @@
-import argparse
-import os
+import warnings
 
+import pytorch_lightning as pl
 import torch
 from config import get_cfg_defaults
-from model import GripNet
-from trainer import Trainer
+from model import GripNetLinkPrediction
+from utils import get_all_dataloader, load_data, setup_supervertex
 
-import kale.utils.logger as lu
 import kale.utils.seed as seed
-from kale.utils.download import download_file_by_url
+from kale.embed.gripnet import GripNet
+from kale.prepdata.supergraph_construct import SuperEdge, SuperGraph, SuperVertex
 
-
-def arg_parse():
-    parser = argparse.ArgumentParser(description="PyTorch CIFAR10 Training")
-    parser.add_argument("--cfg", required=True, help="path to config file", type=str)
-    parser.add_argument("--output", default="default", help="folder to save output", type=str)
-    parser.add_argument("--resume", default="", type=str)
-    args = parser.parse_args()
-    return args
+warnings.filterwarnings(action="ignore")
 
 
 def main():
-    args = arg_parse()
     # ---- setup device ----
     device = "cuda" if torch.cuda.is_available() else "cpu"
-    print("==> Using device " + device)
+    device = torch.device(device)
 
     # ---- setup configs ----
     cfg = get_cfg_defaults()
-    cfg.merge_from_file(args.cfg)
```

Comment:
Good to show example of merging config, see other pykale examples suggested above.